client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -14,15 +14,19 @@ export default class NodeDetailsTable extends React.Component {
 
         {this.props.rows.map(function(row) {
           let valueComponent;
+          let key;
           if (row.value_type === 'numeric') {
+            key = row.key + row.value_major + row.value_minor;
             valueComponent = <NodeDetailsTableRowNumber row={row} />;
           } else if (row.value_type === 'sparkline') {
+            key = row.key;
             valueComponent = <NodeDetailsTableRowSparkline row={row} />;
           } else {
+            key = row.key + row.value_major + row.value_minor;
             valueComponent = <NodeDetailsTableRowValue row={row} />;
           }
           return (
-            <div className="node-details-table-row" key={row.key + row.value_major + row.value_minor}>
+            <div className="node-details-table-row" key={key}>
```

Comment:
Do we need all this stuff in the `key`? `row.key` by itself looks like its unique enough (the label).